### ISSUE TITLE
[BUG 2.0.x]Autobuild STM32F103R_bigtree_USB not detected

### ISSUE THREAD

**Issue by reloxx13** [MarlinFirmware/Marlin#15306]
https://github.com/MarlinFirmware/Marlin/commit/6282655d77d2f9811651843a547b61ca43bd61a9

this is failing somehow.
i get this:
![grafik](https://user-images.githubusercontent.com/14855001/65064491-e0ad3600-d980-11e9-8a9b-b89a17d24cbd.png)

and USB dir is empty. i removed both folders and tried and also cleaned anything.

looks like auto build script is not detecting the right env.

`default_envs = STM32F103R_bigtree_USB`

```
Processing STM32F103R_bigtree (platform: ststm32; framework: arduino; board: genericSTM32F103RC)



Verbose mode can be enabled via `-v, --verbose` option
Mkdir("D:\xampp\htdocs\Marlin\.pio\build\STM32F103R_bigtree")
CONFIGURATION: https://docs.platformio.org/page/boards/ststm32/genericSTM32F103RC.html
PLATFORM: ST STM32 5.6.0 > STM32F103RC (48k RAM. 256k Flash)
HARDWARE: STM32F103RCT6 72MHz, 48KB RAM, 256KB Flash
DEBUG: Current (stlink) External (blackmagic, jlink, stlink)
PACKAGES: tool-stm32duino 1.0.1, toolchain-gccarmnoneeabi 1.70201.0 (7.2.1), framework-arduinoststm32-maple 1.10000.190819 (1.0.0)
Converting Marlin.ino
LDF: Library Dependency Finder -> http://bit.ly/configure-pio-ldf
LDF Modes: Finder ~ chain, Compatibility ~ soft
Found 37 compatible libraries
Scanning dependencies...
Dependency Graph
|-- <U8glib-HAL> 0.4.1
|   |-- <Wire> 1.0
|-- <LiquidCrystal> 1.3.4
|   |-- <Wire> 1.0
|-- <TMCStepper> 0.5.0
|-- <LiquidTWI2>

........


STM32F103R              IGNORED
STM32F103R_fysetc       IGNORED
STM32F103R_bigtree      SUCCESS   00:01:11.661
STM32F103R_bigtree_USB  IGNORED
STM32F4                 IGNORED
STM32F7                 IGNORED
ARMED                   IGNORED
```

myconfig: https://github.com/reloxx13/Marlin/tree/my-skr-mini-e3/Marlin

**Comment by cinealfa** [comment]
Im having the same issue. I had to manually put the bin file on the sd card.

**Comment by anm8tor** [comment]
What is the intention behind the two different  environments? (bigtree vs bigtree_USB)

**Comment by reloxx13** [comment]
mounting sd card over usb (allowing to flash by usb (move fw on sd over usb and trigger a reboot)).

still bugged.

**Comment by boelle** [comment]
@reloxx13  it seems you tried to make a PR 20 days ago? am i right?

**Comment by reloxx13** [comment]
heya,
nope, i dont have a fix for it and its still bugged.
i made a workaround by removing the non-usb version in my platformio.ini.

**Comment by boelle** [comment]
@thinkyhead @shitcreek time to have a look here before the issue gets lost and not fixed

**Comment by PRSRVFRSHNSS** [comment]
Same issue. At this point even when I try to put the firmware.bin on the sd and move it over to the printer the LCD is just blank and unresponsive. Now the only firmware I can get to run on it is the one off the Bigtreetech skr mini e3 master off the github. Can't edit anything and update it without the screen going blank.

**Comment by reloxx13** [comment]
@PRSRVFRSHNSS this is another issue, try reinstalling pio core.
follow this:
https://github.com/MarlinFirmware/Marlin/issues/15767#issuecomment-549156764

**Comment by boelle** [comment]
@thinkyhead @shitcreek time to have a look here before the issue gets lost and not fixed

**Comment by reloxx13** [comment]
looks like fixed with 0f57818f2decca8340fd5fc86f9ea7b792d9d802
